### ISSUE TITLE
Failed to execute 'observe' on 'MutationObserver': parameter 1 is not of type 'Node'.

### ISSUE THREAD

**Issue by calluna18** [kvendrik/intercom-react#29]
I'm getting the following error when using this lib. Is three anything I can do to fix it? 

`Failed to execute 'observe' on 'MutationObserver': parameter 1 is not of type 'Node'.`

Thanks a lot! 🙂

**Comment by ghost** [comment]
@calluna18 Did you find the solution? It is happening to me as well

**Comment by calluna18** [comment]
Nope, unfortunately not, have not had time to look into it @LuisaGDev

**Comment by alluster** [comment]
Also experiencing this problem. Any fixes in the horizon for this?

**Comment by williamisnotdefined** [comment]
The same problem occurs with me.

**Comment by pmoragas** [comment]
Same here

**Comment by StrikerXx798** [comment]
Same here
